velox/functions/sparksql/aggregates/CentralMomentsAggregate.cpp

```diff
@@ -15,51 +15,67 @@
  */
 
 #include "velox/functions/sparksql/aggregates/CentralMomentsAggregate.h"
+#include <limits>
 #include "velox/functions/lib/aggregates/CentralMomentsAggregatesBase.h"
 
 namespace facebook::velox::functions::aggregate::sparksql {
 
 namespace {
+template <bool nullOnDivideByZero>
```

Comment:
I'm not sure why `nullOnDivideByZero` needs to be configurable. Can we just change Spark's implementation?